feat(slm): add three new questions to service level management FAQ (#5428)
newrelic/docs-website: src/content/docs/service-level-management/faqs-slm.mdx

## Can I get alerts on SLI data? [#sli-alerts]

Alerts on New Relic's Service Levels are on our roadmap.

## How does New Relic calculate the remaining error budget? [#error-budget]

The remaining error budget indicates what percentage of requests could still have a bad response over the SLO period without compromising the objective. Therefore, the total amount of tolerated bad responses will vary with the throughput of requests.

Time-based error budgets are on our roadmap.

## How can I view the queries that define an SLI from the UI?

> ![SLI edit menu](./images/slm_edit_menu_2.png "SLI edit menu.png")

Reviewer: I suggest moving the image below the paragraph, just because it's a more usual way to present content.

To view the queries that define an  SLI, you can click on the ... menu on the SLI summary card, and select the Edit option. The queries will show at the top-right corner.

## How can I view the queries that define an SLI through the API?

> To view the queries that define an SLI through the API, you can use [Nerdgraph](https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/). In order to view the SLI configuration, you will need to know the GUID of the entity that the SLI is attached to, and replace it in the following query.

Reviewer: I suggest linking the Service Levels API tutorial directly, instead of the generic intro to nerdgraph. 

```suggestion
To view the queries that define an SLI through the API, you can use [Nerdgraph](https://docs.newrelic.com/docs/apis/nerdgraph/examples/nerdgraph-slm/#retrieve-config). In order to view the SLI configuration, you will need to know the GUID of the entity that the SLI is attached to, and replace it in the following query.
```

```
{
  actor {
    entity(guid: "{entityGuid}") {
      serviceLevel {
        indicators {
          events {
            badEvents {
              where
              from
            }
            goodEvents {
              from
              where
            }
            validEvents {
              from
              where
            }
          }
          name
        }
      }
    }
  }
}
```

> As a response you are going to receive all the SLIs attached to the entity. You’ll get the SLI name and queries. Take into consideration that only one of the goodEvents or the badEvents objects is going to have content,  depending on the configuration of the SLI.

Reviewer: ```suggestion
As a response you are going to receive all the SLIs attached to the entity. You’ll get the SLI name and queries. Take into consideration that only one of the goodEvents or the badEvents objects is going to have content, depending on the configuration of the SLI.
```

> As a response you are going to receive all the SLIs attached to the entity. You’ll get the SLI name and queries. Take into consideration that only one of the goodEvents or the badEvents objects is going to have content,  depending on the configuration of the SLI.

Reviewer: Formatting suggestions:

```suggestion
As a response you are going to receive all the SLIs attached to the entity. You’ll get the SLI name and queries. Take into consideration that only one of the `goodEvents` or the `badEvents` objects is going to have content,  depending on the configuration of the SLI.
```

## Can I chart the SLI attainment on a dashboard?

You can chart SLI attainment time series on your custom dashboards using the following query:

```
FROM Metric SELECT clamp_max((count(newrelic.sli.valid) - count(newrelic.sli.bad)) / count(newrelic.sli.valid) * 100, 100) as 'SLI attainment' WHERE sli.id = '<sli.id>' UNTIL 2 MINUTES AGO TIMESERIES AUTO
```

> Where sli.id is the SLI identifier. The easiest way to add a chart like this to your dashboard is by using the Add to dashboard option, available on the Details view.

Reviewer: ```suggestion
Where `sli.id` is the SLI identifier. The easiest way to add a chart like this to your dashboard is by using the **Add to dashboard** option, available on the Details view.
```

![Add SLI attainment to dashboard](./images/slm_add_to_dashboard.png "Add SLI attainment to dashboard.png")

Alternatively, you can find the SLI id and SLI attainment query through the Nerdgraph API with the following query:

```
{
  actor {
    entity(guid: "{entityGuid}") {
      serviceLevel {
        indicators {
          name
          id
          resultQueries {
            indicator {
              nrql
            }
          }
        }
      }
    }
  }
}
```

> As entityGuid, use the entityGuid of the entity that is associated with the SLI. On the query results, you’ll get the SLI id in the serviceLevel.indicators.id field.

Reviewer: Formatting suggestions:

```suggestion
As `entityGuid`, use the entity GUID of the entity that is associated with the SLI. On the query results, you’ll get the SLI id in the `serviceLevel.indicators.id` field.
```
